src/components/volunteer-enroll/VolunteerEnrollFormWrapper.tsx

```diff
@@ -0,0 +1,137 @@
+    formPages: JSX.Element[];
+};
+
+export const VolunteerEnrolledFormWrapper: React.FC<VolunteerEnrolledPageProps> =
+    ({ session, pageNum, setPageNum, formPages }): JSX.Element => {
+        return (
+            <Wrapper session={session}>
+                {pageNum < formPages.length ? (
+                    <Center>
+                        <Box w={912}>
+                            <Flex
+                                alignItems={"center"}
+                                justifyContent={"space-between"}
+                            >
+                                <BackButton
+                                    onClick={
+                                        pageNum > 0
+                                            ? () =>
+                                                  setPageNum((prevPage) =>
+                                                      Math.max(prevPage - 1, 0),
+                                                  )
+                                            : null
+                                    }
+                                />
+                                <CloseButton />
+                            </Flex>
+                            {formPages.map((formPage, idx) => {
+                                return (
+                                    <Box
+                                        key={idx}
+                                        display={
+                                            pageNum === idx ? null : "none"
+                                        }
+                                    >
+                                        {formPage}
+                                    </Box>
+                                );
+                            })}
+                        </Box>
+                    </Center>
+                ) : (
+                    <Center>
+                        <VStack mt={120} mb={180}>
+                            {/* need to add check icon? doesn't exist on other pages */}
+                            <Text
+                                mb={"55px"}
+                                maxW={476}
+                                fontWeight="700"
+                                fontSize="36px"
+                                align="center"
+                            >
+                                Thank you for signing up to volunteer!
+                            </Text>
+                            <Text maxW={512} textAlign="center">
+                                We're really excited that you want to volunteer
+                                for us. Look out for an email from us shortly
+                                with more information!
+                            </Text>
+                            <Center>
+                                <Link
+                                    _hover={{ textDecoration: "none" }}
+                                    _focus={{}}
+                                    href="/"
```

Comment:
I was thinking this could be redirected to "/class"
```suggestion
                                    href="/class"
```